### ISSUE TITLE
`js-apputils` check fails on CI

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyterlab#13471]
## Description

The `js-apputils` CI check has started failing on multiple PRs recently with the following:

```
Test Suites: 1 failed, 10 passed, 11 total
Tests:       2 failed, 1 skipped, 235 passed, 238 total
Snapshots:   0 total
Time:        31.158 s
Ran all test suites.

Summary of failed tests

test/sessioncontext.spec.ts
  @jupyterlab/apputils
    SessionContext
      .sessionContextDialogs
        #restart()
          ✕ should restart if the user accepts the dialog
          ✕ should not restart if the user rejects the dialog
```

![image](https://user-images.githubusercontent.com/591645/203502930-bb545de2-85ba-4af6-a56f-5003737ec8aa.png)

## Reproduce

This is also reproducible with the latest commit on `master` (6bfef5d478969a6ac59a0ed65405a2df6cdf9109) at the moment: https://github.com/jupyterlab/jupyterlab/actions/runs/3522119316/jobs/5922280054#step:5:22242

## Expected behavior

It should pass.

**Comment by fcollonval** [comment]
This is fixed by https://github.com/jupyter-server/jupyter_server/pull/1088

**Comment by jtpio** [comment]
Nice, for reference this was released in `2.0.0rc7`: https://github.com/jupyter-server/jupyter_server/releases/tag/v2.0.0rc7
